### PR TITLE
Tested with WordPress 6.1

### PR DESCRIPTION
**Pull request by hasnain37**

### CODE REVIEW

**Comment by musamamasood** [comment]
@hasnain37 @afifa-glowlogix please review changes as it revert back to old changes of array syntax.

**Comment by musamamasood** [comment]
Also, resolve conflicts as well.

**Comment by musamamasood** [comment]
https://github.com/glowlogix/wp-frontend-profile/issues/100

**Comment by afifa-glowlogix** [comment]
@hasnain37 kindly run the code through csfixer once.

**Comment by afifa-glowlogix** [comment]
@hasnain37 please resolve conflicts.

File: inc/class-wpfep-profile.php [hunk 1]
```diff
@@ -177,13 +177,12 @@ public function get_profile_url()
...
+            $url = get_permalink($page_id);
+            
+            if (!$url) {
                 return false;
             }
-
-            $url = get_permalink($page_id);
-
-            return apply_filters('wpfep_profile_url', $url, $page_id);
+            return apply_filters('wpfep_profile_url', $url);
```
Comment (musamamasood) [inline inc/class-wpfep-profile.php:185]: @hasnain37 why `$page_id` exclude from `filter`?

**Review by musamamasood — CHANGES_REQUESTED** [review]
Please review comments

**Review by musamamasood — APPROVED** [review]

**Pull request merged**